### PR TITLE
Bug #121252 fix: Newsfeeds: Readmore is not working

### PR DESCRIPTION
**Pull request by Pratikshakhandagale**

### CODE REVIEW

File: easysocial/easysocial/event.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ public function post()
 	 *
 	 * @since 1.0
 	 */
-	public function get_event()
+	private function get_event()
```
Comment (sushan05) [inline easysocial/easysocial/event.php]: @Pratikshakhandagale  - Use camel case

File: easysocial/easysocial/events.php [hunk 2]
```diff
@@ -54,7 +54,7 @@ public function post()
 	 *
 	 * @since 1.0
 	 */
-	public function get_events()
+	private function get_events()
```
Comment (sushan05) [inline easysocial/easysocial/events.php]: @Pratikshakhandagale  - Camel case for method name

File: easysocial/easysocial/events.php [hunk 3]
```diff
@@ -286,7 +285,7 @@ public function dfilter($dates)
 	 *
 	 * @since 1.0
 	 */
-	public function put_status()
+	private function put_status()
```
Comment (sushan05) [inline easysocial/easysocial/events.php]: @Pratikshakhandagale - Camel case for method name

File: easysocial/easysocial/videos.php [hunk 4]
```diff
@@ -60,7 +60,7 @@ public function post()
 	 *
 	 * @since 1.0
 	 */
-	public function get_videos()
+	private function get_videos()
```
Comment (sushan05) [inline easysocial/easysocial/videos.php]: @Pratikshakhandagale  - Camel case

File: easysocial/easysocial/videos.php [hunk 5]
```diff
@@ -157,7 +156,7 @@ public function getVideoDetails($vid = 0)
 	 *
 	 * @since 1.0
 	 */
-	public function upload_videos($type)
+	private function upload_videos($type)
```
Comment (sushan05) [inline easysocial/easysocial/videos.php]: @Pratikshakhandagale  - camelCase

File: easysocial/easysocial/event.php [hunk 6]
```diff
@@ -311,26 +305,26 @@ public function createData($field_ids, $post)
 		$phto_obj = null;
 		$avatar_file_name  = null;
 
-		if (!empty($_FILES['file']['name']))
+		if (!empty($_FILES['avatar']['name']))
```
Comment (thite-amol) [inline easysocial/easysocial/event.php]: @Pratikshakhandagale Use Jinput to retrieve files

File: easysocial/easysocial/event.php [hunk 7]
```diff
@@ -311,26 +305,26 @@ public function createData($field_ids, $post)
...
 		{
 			$upload_obj = new EasySocialApiUploadHelper;
 
 			// Checking upload cover
-			$phto_obj = $upload_obj->ajax_avatar($_FILES['file']);
+			$phto_obj = $upload_obj->ajax_avatar($_FILES['avatar']);
```
Comment (thite-amol) [inline easysocial/easysocial/event.php]: @Pratikshakhandagale same here

**Pull request merged**